build.psm1

```diff
@@ -1513,10 +1512,8 @@ function Install-Dotnet {
         }
 
         if ($uninstallScript) {
-            Start-NativeExecution {
-                curl -sO $obtainUrl/uninstall/$uninstallScript
-                Invoke-Expression "$sudo bash ./$uninstallScript"
-            }
+            Start-NativeExecution ([scriptblock]::Create("curl -sO $obtainUrl/uninstall/$uninstallScript"))
```

Comment:
Using create ScriptBlock is worse that just using a script block.  Please change the curl part back.